bin/_test-helpers.sh

```diff
@@ -333,11 +333,11 @@ run_uninstall_test() {
 
 run_deep_test() {
   local tests=()
-  export helm_path="$bindir"/helm
+  setup_helm
```

Comment:
Sorry, I meant to suggest moving `export helm_path="$bindir"/helm` to `start_test`, not to `setup_helm`. Otherwise calling `setup_helm` here causes building the chart and showing some unwanted output in the (non-helm) deep tests...